web/app/js/components/Sidebar.jsx

```diff
@@ -180,6 +180,13 @@ class Sidebar extends React.Component {
                 : null
             }
 
+            <Menu.Item className="sidebar-menu-item" key="/authorities">
+              <ConduitLink to="/authorities">
+                <Icon type="cloud-o" />
+                <span>Authorities</span>
+              </ConduitLink>
+            </Menu.Item>
```

Comment:
Rather than adding this as a top-level menu item, it feels like it belongs in the resource sub-menu, alongside deployments, replication controllers, and pods.